dpdispatcher/dpcloudserver/config.py

```diff
@@ -0,0 +1,7 @@
+
+HTTP_TIME_OUT = 30
+API_HOST = "http://cloudserver.deepmd.net"
+
+
+ALI_STS_ENDPOINT = 'http://oss-cn-shenzhen.aliyuncs.com'
+ALI_STS_BUCKET_NAME = 'dpcloudserver'
```

Comment:
create dispatcher config all in one is better

Reply:
OK,